core/src/main/scala/com/salesforce/op/stages/impl/classification/MultiClassificationModelSelector.scala

```diff
@@ -49,7 +49,7 @@ case object MultiClassificationModelSelector {
   private[op] val modelNames: Seq[MultiClassClassificationModelsToTry] = Seq(MTT.OpLogisticRegression,
     MTT.OpRandomForestClassifier) // OpDecisionTreeClassifier and OpNaiveBayes off by default
 
-  private val defaultModelsAndParams: Seq[(EstimatorType, Array[ParamMap])] = {
+  private def defaultModelsAndParams: Seq[(EstimatorType, Array[ParamMap])] = {
     val lr = new OpLogisticRegression()
```

Comment:
Is it a duplicate of the code above?
Why not have a trait holding these things?

Reply:
this is not a duplicat they are different levels of control on the models used